soc/xtensa/intel_adsp/ace/include/intel_ace20_lnl/adsp_power.h

```diff
@@ -88,4 +88,14 @@ static ALWAYS_INLINE bool soc_cpu_is_powered(int cpu_num)
 	return (ACE_PWRSTS->dsphpxpgs & BIT(cpu_num)) == BIT(cpu_num);
 }
 
+/**
+ * @brief Retrieve node identifier for Intel ADSP HOST power domain.
+ */
+#define INTEL_ADSP_HST_DOMAIN_DTNODE DT_NODELABEL(hst_domain)
```

Comment:
```suggestion
#define INTEL_ADSP_HOST_DOMAIN_DTNODE DT_NODELABEL(hst_domain)
```

A 3 letters abbreviation for a 4 letters word is pure obfuscation.

Reply:
Thank you for your feedback. The abbreviation 'HST' is used consistently throughout the codebase and documentation to refer to the 'HOST' power domain as per the official naming convention of the domain within the ACE IP architecture. This naming is not intended as obfuscation but rather to maintain alignment with the established terminology that developers and users of the ACE IP are accustomed to. Changing 'HST' to 'HOST' in the code could potentially lead to confusion and discrepancies with other parts of the documentation and the hardware specification. Therefore, I would recommend retaining the 'HST' abbreviation for clarity and consistency across all related materials.

Reply:
Yeah, register docs are register docs and not API choices.  If the hardware folks called a component something ridiculous like "DfPMCCH"[1], then it's our job to transcribe it directly so no one gets confused, even it's it's awful.

[1] Just an example for elucidation.  **Clearly** no reasonable hardware vendor would name an actual register block DfPMCCH.  Right?

Reply:
To be clear though: I think we all agree "HST" is a bad name.  It's just not our name to fix.  It was set as the name of a hardware IP component by designers who have long since moved on to other projects, on products that have long since taped out, and that will just never be changed.  We're stuck with it.